doc/firstWeek summary.md

```diff
@@ -0,0 +1,25 @@
+## First week summary with question and answers.
+
+### Question- Why OAM users needs GItOps? 
+
+> OAM users needs GitOps because of multiple reasons some of them listed below -
+- <b>Deploy Faster More Often</b> When we say “high velocity” we mean that every product team can safely ship updates many times a day — deploy instantly, observe the results in real time, and use this feedback to roll forward or back.
```

Comment:
Why GitOps makes deployment faster? What work does GitOps help save?

> ship updates many times a day — deploy instantly, observe the results in real time, and use this feedback to roll forward or back.

These arguments do not seem to have any relationship to GitOps.